.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "plugins": [
+    "prettier"
+  ],
+  "extends": [
+    "airbnb-base",
+    "plugin:prettier/recommended",
+    "prettier"
+  ],
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true
+  },
+  "rules": {
```

Comment:
If object doesn't have any properties (empty object) just `{}` is enough, no need to add empty lines, the same goes for empty blocks

Reply:
Okay